### PR TITLE
[frontend] - feature(platform services): add background image for openaev and use gradient backgrounds #966

### PR DESCRIPTION
**Pull request by carinelebas**
# Context
> Briefly describe the purpose and context of this PR. Include relevant screenshots or screen recordings for the product team.

We need to add a background image for open aev, with a gradient background. At the same time, use a gradient background for opencti. 

marketing figma: https://www.figma.com/design/Pz1bS978HZ1CsWyv9GYYBk/Thumbnails-Template?node-id=1895-4992&m=dev

result: 

image

Register opencti and an open aev platforms. Check the display.

## Related issues

- Related to #966

# Checklist

## Quality

- [x] I consider this work complete and ready for review
- [x] I tested all features and edge cases
- [x] I refactored code where necessary to improve quality
- [x] I made sure my code meets development guidelines
- [x] I performed a self-review of my code

## Testing

- [ ] I wrote test cases for the relevant use cases (unit, integration or e2e tests)
- [ ] (Bug fixes only) I added tests that reproduce and verify the fix

### What tests have been made

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [x] Local manual tests

# Additional information
> Any additional context, dependencies, or concerns reviewers should know about.

### CODE REVIEW

File: apps/portal-front/src/utils/services.ts [hunk 1]
```diff
@@ -28,6 +28,16 @@ export const isRegistrationService = (
 export const registeredPlatformToServiceInstanceCardData = (
   platform: registerRegisteredPlatformListFragment$data['registeredPlatforms'][number]
 ): ServiceInstanceCardData => {
+  const card_background_by_service_map: Partial<
```
[the remark this answers is not in the document]
Comment (carinelebas) [reply apps/portal-front/src/utils/services.ts]: done in [3387f67](https://github.com/FiligranHQ/xtm-hub/pull/973/commits/3387f67b27d459a1aeedf70204b944ea1a37b19c)
